### PR TITLE
feat(Lockbox): Resolving IOS-1251 - Now showing lockbox side menu item.

### PR DESCRIPTION
**Pull request by thisisalexmcgregor**
## Objective

The `Lockbox` menu item is now visible.

## Description

I added a `.lockbox` menu item. Also, `SideMenuCell` now has a corresponding xib. If the menu item is not new, the `newContainer` will be hidden. 

## How to Test

1. Build and run
2. Go to the menu
3. See that `Lockbox` is shown along with the `new` label.
4. See that the icons have also been updated. 

## Screenshot/Design assessment

simulator screen shot - iphone 8 plus - 2018-10-10 at 15 01 00

Only other thing I noticed is the peek amount seems to be a bit larger in the comps but, if we want to update that I can do that in a separate PR. 

## Merge Checklist

- [x] The PR uses a title supported by [.changelogrc](https://github.com/blockchain/My-Wallet-V3-iOS/blob/dev/.changelogrc#L6...L69).
- [ ] Areas of technical debt are marked with a `// TICKET:` comment that includes a ticket number.
- [x] All unit tests pass.
- [ ] You have added unit tests.
- [x] You have added sufficient documentation (descriptive comments).
- [x] New files added are within the correct directory. (e.g. if a file is required for unit tests to compile, be sure it is added to the tests target.)

### CODE REVIEW

File: Blockchain/Coordinators/AppCoordinator.swift [hunk 1]
```diff
@@ -222,6 +222,9 @@ extension AppCoordinator: SideMenuViewControllerDelegate {
             handleBuyBitcoin()
         case .exchange:
             handleExchange()
+        case .lockbox:
+            // TODO: Present `Lockbox` screen.
```
Comment (bchrisarriola) [inline Blockchain/Coordinators/AppCoordinator.swift]: Nit: TICKET: IOS-1397 cc @achtenhagen

File: Blockchain/LocalizationConstants.swift [hunk 2]
```diff
@@ -337,6 +337,14 @@ struct LocalizationConstants {
             "Support",
             comment: "Item displayed on the side menu of the app for when the user wants to contact support."
         )
+        static let new = NSLocalizedString(
+            "New",
+            comment: "New tag shown for menu items that are new."
+        )
+        static let lockbox = NSLocalizedString(
+            "Lockbox",
```
Comment (bchrisarriola) [inline Blockchain/LocalizationConstants.swift]: Not sure if this needs to be localized 🤔 Let's leave it here though

File: Blockchain/LocalizationConstants.swift [hunk 3]
```diff
@@ -337,6 +337,14 @@ struct LocalizationConstants {
...
+            comment: "Lockbox menu itme title."
```
Comment (bchrisarriola) [inline Blockchain/LocalizationConstants.swift]: s/itme/item

File: Blockchain/Coordinators/AppCoordinator.swift (hunk 1 above)
Comment (thisisalexmcgregor) [reply Blockchain/Coordinators/AppCoordinator.swift]: done 6ea7259

**Pull request merged**